sms-sender-service/sms_sender.bal

```diff
@@ -0,0 +1,59 @@
+import ballerinax/mysql.driver as _;
+import ballerinax/mysql;
+import ballerina/sql;
+import ballerinax/twilio;
+import ballerina/log;
+import ballerinax/nats;
+import ballerinax/jaeger as _;
+
+configurable string natsUrl = ?;
+
+type DataBaseConfig record {|
+    string host;
+    int port;
+    string user;
+    string password;
+    string database;
+|};
+configurable DataBaseConfig databaseConfig = ?;
+final mysql:Client socialMediaDb = check new (...databaseConfig);
+
+type TwilioConfig record {|
+    string accountSid;
+    string authToken;
+    string messageSenderId;
+|};
+configurable TwilioConfig twilioConfig = ?;
+final twilio:Client twilioClient = check new (config = {
+    twilioAuth: {
+        accountSId: twilioConfig.accountSid,
+        authToken: twilioConfig.authToken
+    }
+});
+
+service "ballerina.social.media" on new nats:Listener(natsUrl) {
+    public function init() returns error? {
+        log:printInfo("SMS sender service started");
+    }
+
+    remote function onMessage(int leaderId) returns error? {
+        check sendSms(leaderId);
+    }
+}
+
+type FollowerInfo record {|
+    @sql:Column {name: "mobile_number"} 
+    string mobileNumber;
+|};
+
+function sendSms(int leaderId) returns error? {
+    stream<FollowerInfo, sql:Error?> followersStream = socialMediaDb->query(`
+            SELECT mobile_number 
+            FROM users JOIN followers ON users.id = followers.follower_id
+            WHERE followers.leader_id = ${leaderId};`);
```

Comment:
Lets do this part on the publishing side. I know it is a bit ineffient but it removes the DB dependency. Lets publish something like the below to the queue.

{
    leaderId: 1111;
    followerNo: 07777662261
}

Reply:
@shafreenAnfar we could do like this:
```sh
{
  leaderId: 1111,
  followerNumbers: ["07777662261", "07777662201"]
}
```

Reply:
Fixed with [e7d50dc8ae595f343a63c99c35e6a973f5df2664](https://github.com/shafreenAnfar/ballerina-social-media/pull/7/commits/e7d50dc8ae595f343a63c99c35e6a973f5df2664)